egs/ami/asr1/run.sh

```diff
@@ -108,6 +108,13 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
 	PROCESSED_AMI_DIR=${AMI_DIR}
     fi
     local/ami_${base_mic}_data_prep.sh ${PROCESSED_AMI_DIR} ${mic}
+    # data augmentation
+    utils/perturb_data_dir_speed.sh 0.9 data/${mic}/train_orig data/ihm_tmp1
```

Comment:
`data/${mic}/train_orig` is first appeared.
I think you need to prepare it.

Reply:
Hi, here `ami_${base_mic}_data_prep.sh` in line 110 automatically create the `data/ihm/train_orig` path when the ${mic} is ihm.

Reply:
I see. Thanks.

Reply:
Also, do you think I need to upload the model files?

Reply:
This is very helpful. I sent an invitation to Google Drive. Please upload your model and modify the link accordingly.